fd-beta-version/src/v1/model/dbdeliveries.ts

```diff
@@ -0,0 +1,41 @@
+import { appdb } from "./appdb";
+
+export class dbDeliveries extends appdb {
+  constructor() {
+    super();
+    this.table = "deliveries";
+    this.uniqueField = "id";
+  }
+
+  async getDeliveryPersonId(): Promise<number> {
+    this.table = "users";
+    this.where = `WHERE userType = 'delivery_person'`;
```

Comment:
create object of the model file
let usersObj = new dbusers();
let users_result = await usersObj.allRecords();

In this way

Reply:
changes done